provider/README.md

```diff
@@ -291,6 +296,118 @@ Without an `event_time` query parameter, `/status_changes` shall return a `400 B
 
 [Top][toc]
 
+
+## Reports
+
+Reports are information that providers can send back to agencies containing aggregated data that is not contained within other MDS endpoints, like counts of special groups of riders.
+
+The authenticated reports endpoint are monthly, historic flat files that may be pre-generated by the provider.
```

Comment:
typo here (reports endpoint _are_ monthly)

Reply:
Thanks I think I will reword it to "The authenticated reports are monthly, historic flat files"